ppinot4py/model/measures.py

```diff
@@ -163,3 +165,26 @@ def __str__(self):
         variables = ", ".join([f"{m} is {self.measure_map[m]}" for m in self.measure_map])
 
         return f"the function {self.function_expression} where {variables}"
+
+class BusinessDuration():
+    
+    def __init__(self, business_start, business_end, weekend_list=[5,6], holiday_list=None, unit_hour='min'):
+        super().__init__()
+        
+        self.business_start = business_start
+        self.business_end = business_end
+        self.weekend_list = weekend_list
+        self.holiday_list = holiday_list
+        self.unit_hour = unit_hour
+
+class LogConfiguration():
```

Comment:
This class (LogConfiguration) should be moved to metrics_computer.py because it makes more sense there.